cmd/lora-gateway-bridge/cmd/root.go

```diff
@@ -37,6 +37,10 @@ func init() {
 	rootCmd.PersistentFlags().String("mqtt-ca-cert", "", "")
 	rootCmd.PersistentFlags().String("mqtt-tls-cert", "", "")
 	rootCmd.PersistentFlags().String("mqtt-tls-key", "", "")
+	rootCmd.PersistentFlags().String("last-will-payload", "", "")
```

Comment:
You can remove these flags as they were only there for backwards compatibility of the old CLI flag interface. There is no reason to support new flags as this will be removed in a next version :-)